index.js

```diff
@@ -3,4 +3,109 @@ function getGreeting (firstName, Surname) {
     return 'Hi, ' + '"' + firstName + ' ' + Surname + '".' + " What's up?";
 }
 const greeting = getGreeting('John', 'Smith');
-console.log(greeting); // Hi, “John Smith”. What's up?
\ No newline at end of file
+console.log(greeting); // Hi, “John Smith”. What's up?
+
+// Implement the isThisMyName function that returns true if provided with your name. Otherwise,
+// return false.
+function isThisMyName(name) {
+    return name === 'Marcin';
+    // if (name === 'Marcin') {
+    //     return true;
+    // } else {
+    //     return false;
+    // }
+}
+let isMyName = isThisMyName('Adam');
+console.log(isMyName);
+isMyName = isThisMyName('Marcin');
+console.log(isMyName);
+
+
+
+// 3. Implement the isThisBoolean function that returns true if provided with a boolean.
+
+function isThisBoolean(bool){
```

Comment:
`bool` not always needs to be a boolean. Please figure out a better name for this argument.